### PR TITLE
Code refactoring

### PR DESCRIPTION
**Pull request by guainumbi**
This PR contains some code refactoring for more consistency and less code duplication. It affects the following topics:
- Breadcrumbs

### CODE REVIEW

**Comment by lislis** [comment]
@guainumbi feel free to ping me if/when you want a review!

**Comment by guainumbi** [comment]
if you have time @lislis that would be lovely! Thanks :) (If not - no worries, just let me know. I think nothing is at risk to break.)

File: db/schema.rb [hunk 1]
```diff
@@ -15,12 +15,12 @@
...
   enable_extension "plpgsql"
 
-  create_table "applications", force: :cascade do |t|
+  create_table "applications", id: :serial, force: :cascade do |t|
     t.string "name"
     t.string "email"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.bigint "event_id"
+    t.integer "event_id"
```
Comment (lislis) [inline db/schema.rb]: The schema has changed, was there a migration that ran but you removed later? If not, could you just delete remove the schema from the PR?

Comment (guainumbi) [reply db/schema.rb]: Ah yes, I ran db:migrate a few times while I was dealing with the wrong database credentials. No new migrations though so I will delete the file. Thank you :)

**Pull request merged**